applications_tests/gls_sharp_navier_stokes_2d/flow_around_non_spheres.prm

```diff
@@ -0,0 +1,158 @@
+# Listing of Parameters
```

Comment:
There is a sphere in this example so I wouldn't call it ``flow_around_non_spheres``, but rather something like ``flow_around_multiple_objects`` or something like this.

Reply:
Good point, I changed it